jekyll/_sass/_instantsearch.scss

```diff
@@ -0,0 +1,74 @@
+.base-input-style {
+  border-radius: 0;
+  padding: 1.25em 1em 1em 2.8em;
+  width: 100%;
+  border: none;
```

Comment:
Does this get a border by default, maybe from their code? I'm a little confused to see this and `border-radius: 0`, since those are the defaults 😆